frontend/src/scenes/funnels/FunnelBarGraph.tsx

```diff
@@ -279,7 +279,7 @@ export function FunnelBarGraph(props: ChartParams): JSX.Element {
         visibleStepsWithConversionMetrics: steps,
         stepReference,
         aggregationTargetLabel,
-        isModalActive,
+        isViewedOnDashboard,
```

Comment:
nit: is this better as `isInDashboardContext`?

Viewed feels like a report rather than a state

Reply:
Ooh yeah I like that. May as well make the change now